src/zui/ZUIUserConfigurableDataGrid/useConfigurableDataGridColumns.ts

```diff
@@ -98,7 +98,7 @@ export default function useConfigurableDataGridColumns(
   };
 }
 
-function loadConfig(storage: StorageBackend, key: string): ColumnConfig {
+export function loadConfig(storage: StorageBackend, key: string): ColumnConfig {
```

Comment:
This is a major red flag. Never export something like this without first fully understanding it (and never approve such exports without understanding whether they're necessary).

There are two red flags here:

1. In general, exporting something that was previously not exported is a red flag in itself. Assume that the person who wrote the code refrained from exporting for a reason. Make sure you understand why, before going against that original design.
2. The name of this module is `useConfigurableDataGridColumns`. It had a single default export by the same name (and some relevant types). This is a common pattern. The name of the module indicates that it should have a single export. Adding another export probably goes against the original author's design, and if it does, there better be a good reason (which requires first understanding that design).

Reply:
Thanks for the feedback! It is really helpful and I will make a note of it so it wouldn't be repeated!